test/setup/equippableSlots.ts

```diff
@@ -150,14 +150,11 @@ async function setupContextForSlots(
   }
 
   async function addAssetsToSoldier(): Promise<void> {
-    await soldierEquip.addAssetEntry(
-      soldierResId,
-      0,
-      base.address,
-      'ipfs:soldier/',
-      [partIdForBody], // Fixed parts
-      [partIdForWeapon, partIdForBackground], // Can receive these
-    );
+    await soldierEquip.addAssetEntry(soldierResId, 0, base.address, 'ipfs:soldier/', [
+      partIdForBody,
+      partIdForWeapon,
+      partIdForBackground,
+    ]);
```

Comment:
Does this improve readability? It's ok, just wondering 🤔

Reply:
It's more evident if they are apart but it's a fair compromise. In any new sample or implementation I will simply include either "Part" or "Slot" on the variable name for the part.